### PR TITLE
New package: DVSTSEOOD.QOps version 1.14.15.0

### PR DESCRIPTION
**Pull request by qops-hub**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/136409)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @qops-hub,

Could you add an PackageUrl?

(Automated response - build 660.)

**Comment by stephengillie** [comment]
Manual Validation ended with the package install causing the Manual Validation's PowerShell window to become closed.

**Comment by qops-hub** [comment]
@microsoft-github-policy-service agree [company="{DVSTS EOOD}"]

**Comment by qops-hub** [comment]
@microsoft-github-policy-service agree company="DVSTS EOOD"

**Review by qops-hub — COMMENTED** [review]
nice

**Comment by qops-hub** [comment]
q

**Review by qops-hub — COMMENTED** [review]
.

File: manifests/d/DVSTSEOOD/QOps/1.14.15.0/DVSTSEOOD.QOps.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: DVSTSEOOD.QOps
+PackageVersion: 1.14.15.0
+PackageLocale: en-US
+Publisher: DVSTS EOOD
+PackageName: QOps
```
Comment (stephengillie) [inline manifests/d/DVSTSEOOD/QOps/1.14.15.0/DVSTSEOOD.QOps.locale.en-US.yaml:8]:
```suggestion
PackageName: QOps
PackageUrl: https://github.com/qops-hub/WinGet
```

**Comment by stephengillie** [comment]
Hi @qops-hub,

> Manual Validation ended with the package install causing the Manual Validation's PowerShell window to become closed.

Is this expected?

**Comment by qops-hub** [comment]
> Hi @qops-hub,
> 
> > Manual Validation ended with the package install causing the Manual Validation's PowerShell window to become closed.
> 
> Is this expected?

yes

**Comment by stephengillie** [comment]
This PR keeps being impacted by my automation, and I apologize for this.

**Comment by stephengillie** [comment]
This package is a set of DLLs which constitute a PowerShell module.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**